workers/tasks/export/pandoc.ts

```diff
@@ -91,7 +93,12 @@ const createYamlMetadataFile = async (pubMetadata: PubMetadata, pandocTarget: st
 			title: communityTitle,
 		},
 		copyright: {
-			text: license.full,
+			text:
+				license.short === 'Copyright'
```

Comment:
I also feel like this should be in a utility function, maybe in `utils/licenses.js`?

Reply:
Not sure the best way to do this, because it's already a bit of a hack. Copyright is the only license that relies on info about the pub and community. So if we turn it into a helper, we're forcing ourselves to always pass that information to it. That might just be the way to do it, but it feels a bit inefficient given that most licenses don't need any additional info.

Reply:
Ended up putting it all into the existing getLicenseBySlug function. See what you think. Could probably make it more typescript-y, although it gets called from places with two different styles of metadata objects, so idk.

Reply:
That's the right call, IMO. The fact that it's a hack makes it even _more_ necessary to encapsulate it in a function where the "hackiness" is contained by contract with the caller. I'd argue that we should take it one step further and write a new `getLicenseForPub` function:

```ts
// Taking the License type to be the object that `getLicenseBySlug` returns...
type RenderedLicense = License & {
    summary?: string;
}

type PubWithCollections = Pub & { 
    collectionPubs: DefinitelyHas<CollectionPub, 'collection'>[];
}

const getLicenseForPub = (pub: PubWithCollections, community: Community): RenderedLicense => {
    // call getPubCopyrightYear and getPublisherString in here...
}
```

There's no getting around the fact that we _do_ need a Community and a Pub in scope to correctly compute a Pub's license now, and while I see the temptation to move that responsibility to the caller of `getLicenseBySlug` like this...

```ts
const pubCopyrightYear = getPubCopyrightYear(pubData as any);
const publisherString = getPublisherString(communityData);
const { link, full, slug, summary } = getLicenseBySlug(
    pubData.licenseSlug,
    publisherString,
    pubCopyrightYear,
)!;
```

I weigh that against two other factors:

1. It seems unlikely we'll ever use `getLicenseBySlug(pub, publisherString, pubCopyrightYear)` to do anything other than this exact computation on the results of `getPubCopyrightYear` and `getPublisherString`. And _this_ block of computation still relies on having a Pub and a Community in scope. So we might as well encapsulate it.
2. Having a Pub and a Community to work with in our codebase is _really_ common. We almost always have these at hand when we need them, from `scopeData` on the server and `usePageContext` on the client.

Happy to get on a call and discuss the implementation of `getLicenseForPub` in more detail (or to do it myself). I'm only being persnickety because I have a strong sense that this will lead to much more predictable behavior down the line.